app/models/physical_ticket.rb

```diff
@@ -0,0 +1,14 @@
+class PhysicalTicket < ActiveRecord::Base
+	belongs_to :user
+	belongs_to :ticket_purchase
+
+	scope :by_user, ->(user) { where(user_id: user.id) }
+
+	def ticket_type
+		ticket_purchase.ticket
+	end
+
+	def conference
```

Comment:
this can be done with a through association, but I would say it is not needed yet.